### DISCUSSION TITLE
Relationships and mongodb array in Laravel

### DISCUSSION THREAD

**Discussion by makssein** [jenssegers/laravel-mongodb#2633] [Q&A]
I have a collection:
```
{
    ...
    users: ["..some objectID", "another objectID"]
    ...
}
```
Can I use relationships to get information about these users from the User model or do I need to make a separate query?

I understand that I can extract the ID and make a separate query using whereIn, but this will be very inefficient especially with pagination when a large amount of data is retrieved

**Comment by GromNaN** [comment]
This is how relationships are implemented. The `string` representation of `ObjectId` is used for compatibility with Eloquent. You should look at [Relationships](https://github.com/mongodb/laravel-mongodb/blob/4.1/docs/eloquent-models.md#relationships).

**Reply by makssein** [reply]
In my question I wrote that I want to use relationships, but I don’t know what kind and how
